### ISSUE TITLE
mcptam install script dependencies for g2o

### ISSUE THREAD

**Issue by arunxdas** [wavelab/mcptam#48]
mcptam_install script should automatically run apt-get install ros-distro-libg2o.  This is currently in the ros install script, but should be in the mcptam install script too, for those who already have ros installed on their system and just want to install mcptam.  Else script build fails

**Comment by stevenwaslander** [comment]
Good point, and if we do so, all non-standard ros dependencies should be added.   I actually think the idea was to have these listed on the instructions (readme and wiki), as we also need ros-desktop, catkin_ws etc.  Either works, but it would be odd just add some of the dependencies.  

List:  ros-$ROS_DISTRO-pcl-ros ros-$ROS_DISTRO-image-transport ros-$ROS_DISTRO-image-transport-plugins ros-$ROS_DISTRO-libg2o
